### PR TITLE
Working raw version

### PR DESCRIPTION
**Pull request by OxY623**

### CODE REVIEW

**Review by MikitaTsiarentsyeu — APPROVED** [review]
okay, here we go with classes for the second task) 

So, classes and OOP are basically the good stuff, and it's very good for you knowing about some syntax and principles about Python implementation of it, BUT you are using it completely wrong and out of context. Any class is a way to model some object and/or some process, a way to create a new data type if you need it. And when dealing with oop problems we're using classes as separate additional tools to enrich our program with data types, not as a multitool, which can do basically anything. By implementing your program this way you violated several very important principles, including the responsibility segregation principle and some basic Python-related "beautiful is better then ugly" principle. It's easy to check whether you need classes for the program or not - just try to rework it without classes and if it looks better and works the same, you know the answer. There is no reason for creating a class for the sake of just creating a class, each such entity should address some specific need in your program.

TL;DR - you overcomplicated the program, your class is just a data storage unit, we can live without it and nothing bad will come out of it. But do not worry, we will discuss the OOP during our lessons in each and every detail, and I will show when and how to use them properly. 

Your score is 100+10.
